openc3/lib/openc3/logs/log_writer.rb

```diff
@@ -291,7 +291,7 @@ def prepare_write(time_nsec_since_epoch, data_length, redis_topic = nil, redis_o
         elsif @cycle_size and ((@file_size + data_length) > @cycle_size)
           Logger.debug("Log writer start new file due to cycle size #{@cycle_size}")
           start_new_file() if allow_new_file
-        elsif @enforce_time_order and @previous_time_nsec_since_epoch and (@previous_time_nsec_since_epoch > time_nsec_since_epoch)
+        elsif process_out_of_order and @enforce_time_order and @previous_time_nsec_since_epoch and (@previous_time_nsec_since_epoch > time_nsec_since_epoch)
```

Comment:
`process_out_of_order and @enforce_time_order` ... which is it?

Reply:
Different purposes.    @enforce_time_order is do we care about time order at all, process_out_of_order is do we care about time order for this entry (we don't care for the non-packet entry types).